swift/ITSClient/Sources/ITSMobility/Observers/RoadAlarm/RoadAlarmChangeObserver.swift

```diff
@@ -0,0 +1,32 @@
+/*
+ * Software Name : ITSClient
+ * SPDX-FileCopyrightText: Copyright (c) Orange SA
+ * SPDX-License-Identifier: MIT
+ *
+ * This software is distributed under the MIT license,
+ * see the "LICENSE.txt" file for more details or https://opensource.org/license/MIT/
+ *
+ * Software description: Swift ITS client.
+ */
+
+import Foundation
+
+/// A protocol that defines a observer that receives events of changed road alarms.
+@ChangeObserverActor
+public protocol RoadAlarmChangeObserver: AnyObject, Sendable {
```

Comment:
Will any implementer have to provide the 4 methods, or can they just provide those they are interested in?

What if someone is only intrerested in the low-level raw DENM, and not by the high-level abstracted events `didCreate`, `didUpdate`, and `didDelete`? Would they still have to provide those in their observer?